programs/ssl/ssl_server2.c

```diff
@@ -3199,7 +3199,7 @@ int main( int argc, char *argv[] )
         {
             mbedtls_printf( " failed\n  ! mbedtls_ssl_tls_prf returned -0x%x\n\n",
                             -ret );
-            goto exit;
+            goto reset;
```

Comment:
Why this change? I don't know if it's correct or not, I don't understand it.

Reply:
ping @RonEld : do you remember why you made this change? The commit message says what was changed but not why.

Reply:
All I remember is that the server showed false positive on the results. It passed even though it should have failed.
I don't exactly remember why I did this change, but I think it was to fix the `ssl-opt.sh: 619: kill: No such process` part of the test. the server finished without a graceful shutdown, If I remember correct.

To elaborate, there were two issues:
1. the test result was a false positive
2. the server process terminated before the test (`ssl-opts/sh`) could;d terminate it

Reply:
Thanks for updating. There's a word (at least) missing from the commit message:

> When `ssl_server2` export key functionality,
> don't exit the server (…)

Did you mean “When the `ssl_server2` export key functionality fails,”? Or something else?

Reply:
While you're at it, in the commit message:

* “sll-opts.sh” → “ssl-opt.sh”
* “for verifying export suceeded” → “to verify if the export succeeded”